Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.6-alpine
 
-RUN apk add --update --no-cache git g++ make libstdc++ gnupg musl-dev && \
```

Comment:
`--updated` is required to run `apk update` before adding packages

Reply:
https://github.com/deepmind/kapitan/pull/58 
seems i messed up the commits this one should be in the prior PR sry :(

we won't need the --update since --no-cache is pulling the packages directly from the net.

tested this in the other PR.

Reply:
Great, thanks :)

Reply:
awesome 👍 
sry for the confusion :)